homeassistant/components/blink/sensor.py

```diff
@@ -49,26 +48,32 @@ async def async_setup_entry(
     hass: HomeAssistant, config: ConfigEntry, async_add_entities: AddEntitiesCallback
 ) -> None:
     """Initialize a Blink sensor."""
-    data = hass.data[DOMAIN][config.entry_id]
+    coordinator: BlinkUpdateCoordinator = hass.data[DOMAIN][config.entry_id]
     entities = [
-        BlinkSensor(data, camera, description)
-        for camera in data.cameras
+        BlinkSensor(coordinator, camera, description)
+        for camera in coordinator.api.cameras
         for description in SENSOR_TYPES
     ]
 
     async_add_entities(entities)
 
 
-class BlinkSensor(SensorEntity):
+class BlinkSensor(CoordinatorEntity[BlinkUpdateCoordinator], SensorEntity):
     """A Blink camera sensor."""
 
     _attr_has_entity_name = True
 
-    def __init__(self, data, camera, description: SensorEntityDescription) -> None:
+    def __init__(
+        self,
+        coordinator: BlinkUpdateCoordinator,
+        camera,
+        description: SensorEntityDescription,
+    ) -> None:
         """Initialize sensors from Blink camera."""
+        super().__init__(coordinator)
         self.entity_description = description
-        self.data = data
-        self._camera = data.cameras[camera]
+        self._attr_name = f"{DOMAIN} {camera}"
```

Comment:
I think we just need to remove this. This wasn't here before and if this should change it should do in another PR

Reply:
Good point, same goes for the binary_sensor.  Strange how these things sneak in...